public/introduce.css

```
@@ -0,0 +1,86 @@
+.sea-img{
+  width: 100vw;
+  height: 200px;
+  object-fit: cover;
+  object-position: center;
+  display: block;
+}
+
+.profile-card{
+  text-align: center;
+  display: grid;
+  justify-content: space-between;
+  grid-template-columns: repeat(5, 1fr);
+}
+
+.profile-image{
+  grid-column: 2;
+  width: 200px;
+  height: 200px;
+  object-fit: cover;
+  object-position: top;
+  border-radius: 50%;
+  display: block;
+  margin-top: 25px;
+  margin-left: 0px;
+}
+
+.profile-text {
+  grid-column: 3/5;
+  text-align: center;
+  width: 100%;
+}
+
+.name {
+  font-size: 50px;
+  margin: 0;
+}
+
+.bio {
+  font-size: 16px;
+}
+
+.id {
+  grid-column: 2;
+  font-size: 30px;
+}
+
+.line1 {
```

```suggestion
hr {
```

ここはタグに対してスタイル適用する感じでもいいと思います。
クラス名からイメージ出来なかったので